crane/cmd.go

```diff
@@ -252,6 +262,11 @@ See the corresponding docker commands for more information.`,
 	cmdLogs.Flags().StringVarP(&options.tail, "tail", "", "all", "Output the specified number of lines at the end of logs")
 	cmdLogs.Flags().BoolVarP(&options.colorize, "colorize", "z", false, "Output the lines with one color per container")
 
+	cmdTmpl.Flags().StringVarP(&options.tmplInput, "tmpl", "t", "", "The input template file")
```

Comment:
Ok, it was simply for avoiding `crane template —template "file.tmpl"`. It seems too redundant.